data/gegno/gegno intro missions.txt

```diff
@@ -75,8 +75,7 @@ mission "First Contact: Gegno Vi"
 					goto about
 				`	(Reject their request.)`
 					goto deny
-			`	You hesitantly take the device and turn to make your way to your ship. Before leaving, you take one last glimpse of the enormous creature in the background. Its head is very reptilian, close to dragon-like, with numerous eyes and a mane of spikes. A few small appendages are on either end of it, almost like a scaly salamander. The various signatures on your radar earlier must have been others of their kind. Glaring at you, then at the creature, the alien utters a single intelligible word in a very deep voice:`
-			`	"Voordt."`
+			`	You hesitantly take the device and turn to make your way to your ship. Before leaving, you take one last glimpse of the enormous creature in the background. Its head is very reptilian, close to dragon-like, with numerous eyes and a mane of spikes. A few small appendages are on either end of it, almost like a scaly salamander. The various signatures on your radar earlier must have been others of their kind. The alien then glares at you, then at the creature, and back to you once more before continuing towards your ship.`
```

Comment:
Reworked this slightly -- the last sentence's flow felt off to me so I tweaked out a `then` and `and`, took out the slightly redundant `very reptilian, close to dragon-like`, and tried to make the salamander-like appendages more clear.

```suggestion
			`	You hesitantly take the device and turn to make your way to your ship. Before leaving, you take one last glimpse of the enormous creature in the background. Its head appears dragon-like, though with numerous eyes and a mane of spikes. A few small appendages sprout from its neck, giving it a scaly, salamander-like appearance. The various signatures on your radar earlier must have been others of their kind. The alien glares at you, then at the creature, before turning back to you once more to continue towards your ship.`
```